catroid/src/main/AndroidManifest.xml

```diff
@@ -209,6 +209,26 @@
             </intent-filter>
         </activity>
         <activity
+            android:name=".ui.AccessibilityPreferencesActivity"
+            android:label="@string/preference_access_title_activity_access"
+            android:noHistory="false"
+            android:screenOrientation="portrait">
+            <intent-filter>
+                <action android:name=".ui.AccessibilityPreferencesActivity" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
```

Comment:
nevermind, I removed the intent action in the xml and did it in the code with a clicklistener instead. intent filters are gone.